### DISCUSSION TITLE
Retry download/Search and download again

### DISCUSSION THREAD

**Discussion by ShivamAmin** [RandomNinjaAtk/docker-lidarr-extended#104] [Q&A] [answered]
Hey, so I've had lidarr-extended running for a few days or so. I've got all the music that I wanted, but I sometimes find tracks that should be explicit but aren't. What I did to fix this is to delete the track in lidarr, then delete everything under config/extended/logs/downloaded/deezer. Is there a better way? Ideally, I'd like to be able to delete the file in lidarr and retry the download. I guess the workaround for this would be to use the Audio Lyric Type, but that would require me to start over again.

**Comment by RandomNinjaAtk** [comment, marked as answer]
Generally, there is no database that I'm aware of that tracks or says if a particular release group should have a explicit release. 

So its not possible to make sure it only downloads explicit releases for a particular release group because there is no centralized database to check against/verify that there should be one available. This applies to both the script and typical usenet/torrent. 

So setting the language restriction will force it to only download the matching type via script. But you'll miss a lot because of it.

I don't believe there is anyway to improve the situation in its current state with the available information....

The script is designed to perform as a best effort approach to always attempt to grab a explicit release first. But there is no guarantee that will happen.
